cmd/flags.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package cmd
+
+import (
+	"github.com/urfave/cli"
+	"log"
```

Comment:
External package should below under go's packages and separate with a blank line.

Reply:
I'll change that.

Could this be added to `make fmt-check`?
When the blank line is missing `make fmt` produces the following change:
```
--- ./cmd/flags.go.orig 2019-05-05 09:56:31.835956980 +0000
+++ ./cmd/flags.go      2019-05-05 09:56:31.835956980 +0000
@@ -5,8 +5,8 @@
 package cmd
 
 import (
-       "log"
        "github.com/urfave/cli"
+       "log"
 )
```